t/x509.t

```diff
@@ -55,6 +55,7 @@ is($$exts{"2.5.29.19"}->object()->name(),"X509v3 Basic Constraints", "Extension-
 
 ok($$exts{"2.5.29.19"}->is_critical(), "basic constraints is critical");
 ok($$exts{"2.5.29.19"}->basicC("ca"), 'basicConstraints CA: TRUE 2.4.1');
+ok($$exts{"2.5.29.19"}->as_string() eq $$exts{"2.5.29.19"}->to_string(), 'as_string is an alias of to_string');
```

Comment:
Please use single-quotes for literals as suggested by [Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals](https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitInterpolationOfLiterals)

```suggestion
ok($$exts{'2.5.29.19'}->as_string() eq $$exts{'2.5.29.19'}->to_string(), 'as_string is an alias of to_string');